.changeset/modern-ghosts-sin.md

```diff
@@ -0,0 +1,37 @@
+---
+"@logto/tunnel": minor
+---
+
+add deploy command and env support
+
+#### Add new `deploy` command to deploy your local custom UI assets to your Logto Cloud tenant
+
+1. Create a machine-to-machine app in your Logto tenant
+2. Run the following command
+
+```bash
+npx @logto/tunnel deploy --auth <your-m2m-app-id>:<your-m2m-app-secret> --endpoint https://<tenant-id>.logto.app --experience-path /path/to/your/custom/ui
+```
+
+If you are using custom domain, an additional option `--management-api-resource` (or `--resource`) must be specified.
```

Comment:
should we also mention that using custom domain has the same effect? i.e. it doesn't matter to use which endpoint for uploading

Reply:
You mean default domain has the same effect?

Reply:
updated